src/Workleap.Extensions.Mongo.Tests/MongoIndexerTests.MultipleIndexers.cs

```diff
@@ -0,0 +1,135 @@
+using Microsoft.Extensions.DependencyInjection;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Driver;
+using Workleap.Extensions.Mongo.Indexing;
+
+namespace Workleap.Extensions.Mongo.Tests;
+
+public partial class MongoIndexerTests
```

Comment:
Why not just put them in the existing class?

Reply:
Or if we want to keep them seperate, give it another name

Reply:
Maybe it does! Just not a fan of splitting tests into many partial classes, what happens when there's 4 partial classes for the same class